dataset_builders/hf/README.md

```diff
@@ -0,0 +1,128 @@
+# Dataset Card for "CDCP"
+
+### Dataset Summary
+
+CDCP (a.k.a. *Cornell eRulemaking Corpus*; [Park and Cardie, 2018](https://aclanthology.org/L18-1257.pdf)) consists of 731 user comments from an eRulemaking platform in the English language. There are five types of components (`Fact`, `Testimony`, `Reference`, `Value`, and `Policy`) and two types of supporting relations (`Reason` and `Evidence`) are annotated on the basis of the study by Park et al. (2015). The resulting dataset contains 4931 elementary unit and 1221 support relation annotations. (pp. 1623-1624)
+
+### Supported Tasks and Leaderboards
+
+- **Tasks:** Argument Minning, Link Prediction, Component Classification, Relation Classification
+- **Leaderboards:** https://paperswithcode.com/dataset/cdcp
+
+### Languages
+
+The language in the dataset is English (AmE).
+
+## Dataset Structure
+
+### Data Instances
+
+- **Size of downloaded dataset files:** 5.37 MB
+
+```
+{
+  'id': "00195",
+  'text': "State and local court rules sometimes make default judgments much more likely. For example, when a person who allegedly owes a debt is told to come to court on a work day, they may be forced to choose between a default judgment and their job. I urge the CFPB to find practices that involve scheduling hearings at inconvenient times unfair, deceptive, and abusive, or inconsistent with 1692i",
+  'proposition': {
+        "start": [0, 78, 242],
+        "end": [78, 242, 391],
+        "label": [4, 4, 1],
+        "url": ["", "", ""],
+    },
+  'relations': {"head": [0, 2], "tail": [1, 0], "label": [1, 1]},
+}
+```
+
+### Data Fields
+
+- `id`: the instance id of the text, a `string` feature
+- `text`: the text, a `string` feature
+- `proposition`: the annotation list of spans with labels and URL (if applicable), a `dictionary` feature
```

Comment:
Can you explain what `url` is in a bit more detail? If I remember correctly,  they did replace them in the `text` or something?

EDIT: Please add subitems for `start`, `end`, `label` , and `url` below `propositions` and describe them shortly (start / end are indices into the text, is end inclusive or exclusive? what is url, to what labels do the label indices refer to?). Also for the relations (what are the numbers?).